addons/calendar/wizard/calendar_popover_delete_wizard.py

```diff
@@ -1,25 +1,80 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class CalendarProviderConfig(models.TransientModel):
     _name = 'calendar.popover.delete.wizard'
+    _inherit = 'mail.composer.mixin'
     _description = 'Calendar Popover Delete Wizard'
 
 
     record = fields.Many2one('calendar.event', 'Calendar Event')
     delete = fields.Selection([('one', 'Delete this event'), ('next', 'Delete this and following events'), ('all', 'Delete all the events')], default='one')
+    recipient_ids = fields.Many2many(
+        'res.partner',
+        string="Recipients",
+        compute='_compute_recipient_ids',
+        readonly=False,
+    )
 
     def close(self):
-        if not self.record or not self.delete:
-            pass
-        elif self.delete == 'one':
-            self.record.unlink()
+        return self.record.action_unlink_event(self.record.partner_id.id, self.delete)
+
+    @api.depends('record')
+    def _compute_recipient_ids(self):
+        for wizard in self:
+            wizard.recipient_ids = wizard.record.partner_id | wizard.record.message_partner_ids
+
+    @api.depends('record')
+    def _compute_subject(self):
+        for wizard_subject in self.filtered('template_id'):
+            wizard_subject.subject = wizard_subject.template_id._render_field(
+                'subject',
+                [wizard_subject.record.id],
+                compute_lang=True,
+                options={'post_process': True},
+            )[wizard_subject.record.id]
+
+    @api.depends('record')
+    def _compute_body(self):
+        for wizard_body in self.filtered('template_id'):
```

Comment:
```suggestion
        for wizard in self.filtered('template_id'):
```
```suggestion
        for wizard in self.filtered('template_id'):
```
This seems better, because currently we are accessing `wizard_body.body` in the following loop.